site/documentation/content/api/event/index.md

```diff
@@ -88,10 +88,14 @@ The AMQP 1.0 properties an event sender needs to set for an *empty notification*
 
 The relevant properties are listed again in the following table:
 
-| Name           | Mandatory        | Location                 | Type      | Description |
-| :------------- | :--------------: | :----------------------- | :-------- | :---------- |
-| *content-type* | yes              | *properties*             | *symbol*  | MUST be set to *application/vnd.eclipse-hono-empty-notification* |
-| *ttd*          | no               | *application-properties* | *int*     | The *time 'til disconnect* as described in the [*Telemetry API*]({{< relref "/api/telemetry" >}}). |
+| Name                       | Mandatory        | Location                 | Type      | Description |
+| :------------------------- | :--------------: | :----------------------- | :-------- | :---------- |
+| *content-type*             | yes              | *properties*             | *symbol*  | MUST be set to *application/vnd.eclipse-hono-empty-notification* |
+| *ttd*                      | no               | *application-properties* | *int*     | The *time 'til disconnect* as described in the [*Telemetry API*]({{< relref "/api/telemetry" >}}). |
+| *hono_registration_status* | no               | *application-properties* | *string*  | If set to `NEW` this indicates that a device was auto-provisioned in the course of sending telemetry data via gateway. |
+| *tenant_id*                | no               | *application-properties* | *string*  | The tenant id denoting the tenant for which the device was auto-provisioned. |
+| *device_id*                | no               | *application-properties* | *string*  | The device id denoting of the auto-provisioned device. |
+| *gateway_id*               | no               | *application-properties* | *string*  | The gateway id denoting the gateway for which the device was auto-provisioned. |
```

Comment:
Is there a particular reason why we are (ab)using the empty notification for reporting the provisioning of a device? Why don't we use a specific event with its own content-type for this purpose. Wouldn't that make processing of the event easier?

Reply:
Well, I guess at the time we discussed we thought that it is an acceptable approach (see: https://github.com/eclipse/hono/pull/2094#issuecomment-668473083).

I didn't hear any concerns from the Ditto team (which is the only client of this event sub type to my knowledge) regarding the complexity of the processing of the event. Nevertheless it is right that clients would save one conditional if we had used a dedicated event.

Nevertheless I don't think that it is significantly more complex with the empty event...

Reply:
> Well, I guess at the time we discussed we thought that it is an acceptable approach (see: #2094 (comment)).

Now I see the problem. There seems to have been a misunderstanding on my side regarding what you were planning to do. My understanding was that we will add a *new type of* event which will have no payload, hence an *empty event* but with its own *content-type*, hence my answer
> However, we should add this particular event to the Well known events ...

FMPOV the chosen approach is inconsistent with the way we have designed the *well known events*, which is to indicate the purpose of the event by means of the content-type and then define the properties according to the purpose.
Now we re doing it the other way around: by inspecting the presence/absence of particular properties, we need to deduce the purpose of the event :-(. I would like this to be changed so that we have a dedicated event (type) for indicating that a device has been provisioned. But we should do this in another PR ... I have created #2408 for tracking this.

Reply:
> Now I see the problem. There seems to have been a misunderstanding on my side regarding what you were planning to do. My understanding was that we will add a new type of event which will have no payload, hence an empty event but with its own content-type, hence my answer

Sorry for that 😞, it would have been better if I had written "Empty Notification" instead but I mixed that one up with "Empty Event"...

Reply:
No worries, we'll get it fixed ...